addons/mail/tests/test_mail_render.py

```diff
@@ -53,6 +53,13 @@ def setUpClass(cls):
 </p>"""
         ]
 
+        # some templates with local links to replace (see _replace_local_links)
+        cls.base_local_links_template_bits = [
```

Comment:
Actually this could be defined directly in ``test_render_template_local_links`` test (no need to bloat cls with data used only once :) )